src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -490,7 +497,6 @@ def determine_needed_experience_configs(bind):
                     config["accept_button_label"] = eec["accept_button_label"]
                     config["acknowledge_button_label"] = eec["acknowledge_button_label"]
                     config["banner_description"] = eec["banner_description"]
-                    config["banner_enabled"] = eec["banner_enabled"]
```

Comment:
Banner enabled is a field we're now keeping for auditing purposes on older historical records only. But newly created records through this migration should not have this value set.